### PR TITLE
fix(frontend): The "Follow" / "Following" buttons on user components are not updated on the new tab after following / unfollowing the users tm-444

### PR DESCRIPTION
**Pull request by GEOFARL**

### CODE REVIEW

File: apps/frontend/src/pages/friends/friends.tsx [hunk 1]
```diff
@@ -79,6 +79,12 @@ const Friends: React.FC = () => {
 						page: potentialFriendsPagination.page,
 					}),
 				);
+				void dispatch(
+					actions.getFollowings({
+						count: PaginationValue.DEFAULT_COUNT,
+						page: followingsPagination.page,
+					}),
+				);
```
Comment (GEOFARL) [inline apps/frontend/src/pages/friends/friends.tsx:87]: Haven't found a better way with current implementation to update a stale followings data. We do not handle it client side as we do not want to remove friend cards after following or unfollowing them

Comment (what1s1ove) [reply apps/frontend/src/pages/friends/friends.tsx:87]: you need t refetch friends on the each page after opening all the time. and you need to fetch friends for the opened page only

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: apps/frontend/src/modules/friends/slices/friends.slice.ts [hunk 2]
```diff
@@ -108,6 +110,24 @@ const { actions, name, reducer } = createSlice({
...
+			state.locallyUnfollowedFriendIds.push(action.payload);
+		},
+		clearFollowings(state) {
+			state.followings = state.followings.filter((following) => {
+				return !state.locallyUnfollowedFriendIds.includes(following.id);
+			});
+		},
+		removeLocallyUnfollowedFriendId(state, action: PayloadAction<number>) {
+			state.locallyUnfollowedFriendIds =
+				state.locallyUnfollowedFriendIds.filter((id) => id !== action.payload);
```
Comment (ShadowOfTheSpace) [inline apps/frontend/src/modules/friends/slices/friends.slice.ts:129]: B27 here

**Review by ShadowOfTheSpace — CHANGES_REQUESTED** [review]

File: apps/frontend/src/modules/friends/slices/friends.slice.ts (hunk 2 above)
Comment (Oleksandra-Masiuk) [reply apps/frontend/src/modules/friends/slices/friends.slice.ts:129]: @ShadowOfTheSpace 
image

Comment (ShadowOfTheSpace) [reply apps/frontend/src/modules/friends/slices/friends.slice.ts:129]: @what1s1ove said on one of the daily calls that we will not use that exception for our project right now, so even one-line arrow functions need to have curly braces

**Review by Sytnikovakatya — APPROVED** [review]

File: apps/frontend/src/modules/friends/slices/friends.slice.ts (hunk 2 above)
Comment (what1s1ove) [reply apps/frontend/src/modules/friends/slices/friends.slice.ts:129]: it's okay for only the one line function

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**